tests/NuGet.Services.V3.Tests/Support/InMemoryCloudBlobClient.cs

```diff
@@ -5,13 +5,13 @@
 using System.Collections.Generic;
 using NuGetGallery;
 
-namespace NuGet.Services.AzureSearch.Catalog2AzureSearch.Integration
+namespace NuGet.Services
 {
     public class InMemoryCloudBlobClient : ICloudBlobClient
     {
         private readonly object _lock = new object();
 
-        public Dictionary<string, InMemoryCloudBlobContainer> Containers { get; } = new Dictionary<string, InMemoryCloudBlobContainer>();
+        public SortedDictionary<string, InMemoryCloudBlobContainer> Containers { get; } = new SortedDictionary<string, InMemoryCloudBlobContainer>();
```

Comment:
Why `SortedDictionary`? I'm just curious, LGTM 😊

Reply:
Makes assertions of expected keys easier. Otherwise you have to sort the "actual" as well as hard coded sorted "expected".